torchvision/transforms/v2/_misc.py

```diff
@@ -278,10 +280,27 @@ def __init__(
                 f"Got {labels_getter} of type {type(labels_getter)}."
             )
 
+    @staticmethod
+    def _get_dict_or_second_tuple_entry(inputs):
+        # Often, datasets outputs are plain dicts like {"img": ..., "labels": ..., "bbox": ...}
+        # But they can also be tuples (img, {"labels":..., "bbox": ...})
+        # This hacky helper accounts for both structures.
+        if isinstance(inputs, tuple):
+            inputs = inputs[1]
+
+        if not isinstance(inputs, collections.abc.Mapping):
+            raise ValueError(
+                f"If labels_getter is a str or 'default', "
+                f"then the input to forward() must be a dict or a tuple whose second element is a dict."
+                f" Got {type(inputs)} instead."
```

Comment:
I acknowledge `type(inputs)` is only partially useful in the message if the input is a tuple, as we don't say what is the type of `tuple[1]`... Do we care?

Reply:
Not sure I understand. 

> as we don't say what is the type of `tuple[1]`... Do we care?

We do exactly that since you did `inputs = inputs[1]`. Am I missing something here?

Reply:
ah, true!